backend/models/organizations/orgs-model.test.js

```diff
@@ -11,7 +22,107 @@ afterAll(async () => await db.closeDatabase())
-    test("Dummy test", () => {
-        expect(true).toBe(true)
+    test("Database Has Organizations Model", () => {
+        expect(OrgsModel).toBeDefined()
+    })
+
+    describe("Organization Test Series", () => {
+        test("Get Organization Instance from Database", async () => {
+            const org = new OrgsModel(testOrg1)
+            await org.save()
+            const retrievedOrg = await OrgsModel.findOne({
+                name: "Hackathon",
+            })
+
+            // Expected value set
+            const expectedName = testOrg1.name
+            const expectedDescription = testOrg1.description
+            const expectedUrl = testOrg1.url
+            const expectedTags = testOrg1.tags
+            const expectedCreatedDate = testOrg1.created_at
+            const expectedUpdatedDate = testOrg1.updated_at
+
+            // Tests
+            expect(retrievedOrg).toBeDefined()
+            expect(retrievedOrg.name).toEqual(expectedName)
+            expect(retrievedOrg.description).toEqual(expectedDescription)
+            expect(retrievedOrg.url).toEqual(expectedUrl)
+            expect(retrievedOrg.tags).toEqual(expectedTags)
+            expect(retrievedOrg.created_at).toEqual(expectedCreatedDate)
+            expect(retrievedOrg.updated_at).toEqual(expectedUpdatedDate)
+        })
+
+        test("Add an Organization to Database", async () => {
+            const org = new OrgsModel(testOrg1)
+
+            const createdOrg = await org.save()
+
+            // Expected value set
+            const expectedName = testOrg1.name
+            const expectedDescription = testOrg1.description
+            const expectedUrl = testOrg1.url
+            const expectedTags = testOrg1.tags
+            const expectedCreatedDate = testOrg1.created_at
+            const expectedUpdatedDate = testOrg1.updated_at
+
+            // Tests
+            expect(createdOrg._id).toBeDefined()
+            expect(createdOrg.name).toEqual(expectedName)
+            expect(createdOrg.description).toEqual(expectedDescription)
+            expect(createdOrg.url).toEqual(expectedUrl)
+            expect(createdOrg.tags).toEqual(expectedTags)
+            expect(createdOrg.created_at).toEqual(expectedCreatedDate)
+            expect(createdOrg.updated_at).toEqual(expectedUpdatedDate)
+        })
+
+        test("Update an Organization in Database", async () => {
+            const org = new OrgsModel(testOrg1)
+            await org.save()
+
+            // Expected value set
+            const updatedName = "Hackathon Hyperdrive"
+            const updatedDescription = "Hackathon Hyperdrive is a hackathon"
+            const updatedUrl = "https://www.hackathonhyperdrive.com"
+            const updatedTags = [
+                "hackathon",
+                "coding",
+                "software",
+                "hyperdrive",
+            ]
+            const updatedUpdatedDate = new Date("09-30-2022")
```

Comment:
Either remove this line or add a different date. This can cause a false pass test.